src/test/java/org/assertj/core/condition/VerboseConditionTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2021 the original author or authors.
+ */
+package org.assertj.core.condition;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
+
+import org.assertj.core.api.BDDAssertions;
+import org.assertj.core.api.Condition;
+import org.junit.jupiter.api.Test;
+
+class VerboseConditionTest {
```

Comment:
replace `assertThat` by `then` as indicated in the contributing guidelines

Reply:
done, bit did not found an equivalent so `assertThatNullPointerException`